src/components/NotSupported/NotSupported.js

```diff
@@ -1,26 +1,23 @@
 import React from 'react';
 import * as PropTypes from 'prop-types';
-import P from '../P';
+import Paragraph from '../Paragraph';
+import { network } from '../../config';
 
 const NotSupported = ({ onclick }) => {
-  const host = window.location.hostname;
-  const network =
-    host.includes('local') || host.includes('dev') || host.includes('test')
```

Comment:
weird flex but ok

idk, for me, seems very simple stupid hoping for hostname, but if it works - why not.

Seems like you don't have normal separation between development and production, but this is more complex problem then just choosing between two names in popup.

Reply:
yes, you correctly noted, this is because of the API metamask.
here you need to think more asynchronously ...
The line between test and pod environments is palpable